# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/NeckoChild.h
# Commit: da41f0fab427
# Full Hash: da41f0fab427b199f1a0c6156467a0ad9a9e6891
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Make proxy config parameters optional in WebrtcTCPSocket to get the NAT simulator working again. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45104
# ==============================================================================

diff -r b83124cf3a89 -r da41f0fab427 netwerk/ipc/NeckoChild.h
--- a/netwerk/ipc/NeckoChild.h	Tue Sep 17 18:17:24 2019 +0000
+++ b/netwerk/ipc/NeckoChild.h	Tue Sep 17 18:17:37 2019 +0000
@@ -28,7 +28,7 @@
   PStunAddrsRequestChild* AllocPStunAddrsRequestChild();
   bool DeallocPStunAddrsRequestChild(PStunAddrsRequestChild* aActor);
 
-  PWebrtcTCPSocketChild* AllocPWebrtcTCPSocketChild(const TabId& tabId);
+  PWebrtcTCPSocketChild* AllocPWebrtcTCPSocketChild(const Maybe<TabId>& tabId);
   bool DeallocPWebrtcTCPSocketChild(PWebrtcTCPSocketChild* aActor);
 
   PAltDataOutputStreamChild* AllocPAltDataOutputStreamChild(